src/main/java/me/pikamug/localelib/LocaleManager.java

```diff
@@ -36,9 +42,11 @@ public LocaleManager() {
 		}
 		String version = Bukkit.getServer().getClass().getPackage().getName().split("\\.")[3];
 	    try {
+	    	Class<?> localeClass = Class.forName("net.minecraft.server.{v}.LocaleLanguage".replace("{v}", version));
+            locale = localeClass.newInstance();

```

Comment:
Why create a new instance? It's my understanding that this is bad practice. See https://stackoverflow.com/questions/195321/why-is-class-newinstance-evil

Reply:
Its because I don't know any working alternative. Its implemented in Java for a purpose and since its not deprecated, we can still use it.
But please let me know if you can find any alternative to this that works and is more commonly used.

Reply:
I just don't see the need to create a new instance. In fact, it doesn't seem like we need a separate object at all, but perhaps I'm wrong.